src/FSharpPlus/Validations.fs

```diff
@@ -113,6 +113,7 @@ module Validation=
     ///
     /// validate : 'e -> ('a -> bool) -> 'a -> Validation<'e, 'a>
     ///
+    [<System.Obsolete("The validate method is less generic than intended.")>]
```

Comment:
I don't think we should give that reason for the deprecation. Let's put something like "This function will not be supported in future versions."

Reply:
Let's go with that then